WORKSPACE

```diff
@@ -112,29 +112,24 @@ maven_install(
         "org.robolectric:annotations:4.3",
         "androidx.appcompat:appcompat:1.0.2",
         "androidx.annotation:annotation:1.1.0",
-        "com.android.support:support-annotations:28.0.0",
-        "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.2",
-        "org.jetbrains.kotlinx:kotlinx-coroutines-test:1.3.8",
         "androidx.core:core-ktx:1.0.1",
-        "junit:junit:4.12",
-        "com.google.truth:truth:0.43",
-        "com.github.bumptech.glide:glide:4.11.0",
-        "com.github.bumptech.glide:compiler:4.11.0",
-        "com.caverock:androidsvg-aar:1.4",
-        "com.crashlytics.sdk.android:crashlytics:2.9.8",
-        "io.fabric.sdk.android:fabric:1.4.7",
-        "com.google.gms:google-services:4.3.3",
         "androidx.lifecycle:lifecycle-livedata-ktx:2.2.0-alpha03",
         "androidx.test.ext:junit:1.1.1",
         "com.android.support:support-annotations:28.0.0",
         "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.2",
         "org.jetbrains.kotlinx:kotlinx-coroutines-test:1.2.2",
         "org.jetbrains.kotlin:kotlin-stdlib-jdk7:jar:1.3.72",
         "org.jetbrains.kotlin:kotlin-test-junit:1.3.72",
+        "org.jetbrains.kotlin:kotlin-reflect:1.3.41",
         "com.google.truth:truth:0.43",
         "com.google.firebase:firebase-analytics:17.4.4",
         "com.google.firebase:firebase-crashlytics:17.1.1",
-        "org.jetbrains.kotlin:kotlin-reflect:1.3.41",
+        "com.crashlytics.sdk.android:crashlytics:2.9.8",
+        "io.fabric.sdk.android:fabric:1.4.7",
```

Comment:
Nit: please alphabetize these & all other deps lists when you update them.

Reply:
On it!